papermill/tests/test_execute.py

```diff
@@ -14,11 +16,14 @@
 from . import get_notebook_path, RedirectOutput
 
 python_2 = sys.version_info[0] == 2
+PYTHON = 'python2' if python_2 else 'python3'
+execute_notebook = partial(execute_notebook, kernel_name=PYTHON)
```

Comment:
I like this solution because I ❤️ `partial` ... I could be persuaded that this is trying to be too clever (for tests which should be as simple as possible). WDYT?

Reply:
This is perfect, I'm happy with it as a functional programming proponent.